src/SystemView.h

```diff
@@ -80,10 +80,10 @@ struct Projectable {
 class SystemView : public PiGuiView, public DeleteEmitter {
 public:
 	SystemView(Game *game);
-	virtual ~SystemView();
-	virtual void Update();
-	virtual void Draw3D();
-	// virtual void OnSwitchTo() { Update(); Draw3D(); }
+	~SystemView() override;
+	void Update() override;
+	void Draw3D() override;
+	void OnSwitchFrom() override { m_projected.clear(); } // because ships from the previous system may remain after last update
```

Comment:
I would recommend this go in the .cpp file instead of the header just to keep things clean. SystemView.h is already pretty big.

Reply:
Done!